TFNetworkLayer.py

```diff
@@ -1034,6 +1037,7 @@ def batch_norm(self, data,
     :param tf.Tensor sample_variance:
     :param tf.Tensor gamma:
     :param tf.Tensor beta:
+    :param bool fused: use fused version of batch normalisation
```

Comment:
Why do you have this as an option? Or rather, why is the default `False`? I thought that you made sure via the tests that it does not matter at all in behavior? In that case, why can't you always have this `True`?

Reply:
It's false to make the old version default.

Reply:
I thought they are equivalent? If they are equivalent, why is there any reason to not use the fused version if possible? So shouldn't it be the default then (if it is possible to use)? (Maybe use `None`, or `NotSpecified` as the default value, and then in the code automatically set it to `True` if it is possible to use.)
If they are not equivalent, then this is wrong and should be fixed. It should be equivalent.

Reply:
I see your point, I'll do this way

Reply:
> I thought they are equivalent? If they are equivalent, why is there any reason to not use the fused version if possible? So shouldn't it be the default then (if it is possible to use)? (Maybe use `None`, or `NotSpecified` as the default value, and then in the code automatically set it to `True` if it is possible to use.)
> If they are not equivalent, then this is wrong and should be fixed. It should be equivalent.

I think it's better keep the default as `False`. This way, when loading old configs, the nonfused version is used. Otherwise, old checkpoints can't be loaded into fused variables.

Reply:
But then this is a bug. Old checkpoints should be able to be loaded, also when you have `fused=True` here. This is what I explained earlier. And also you should have a test case for this (as I also explained earlier).